spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -10,7 +10,7 @@
 
 package: swiftnav.sbp.ssr
 description: Precise State Space Representation (SSR) corrections format
-stable: False
+stable: True
```

Comment:
@jbangelo Should we mark this as stable?

Reply:
I think so. There might be changes in the future, but we don't have anything planned in the foreseeable future and these messages have been doing well so far.